### PR TITLE
Games page progress section fix

### PR DESCRIPTION
**Pull request by tmansuroglu**
Progress section functionality couldn't be delivered on time. Applied placeholder progress data. Also, Grid layout made adaptive to Arabic. #94

### CODE REVIEW

**Review by majdajroudi — APPROVED** [review]

**Comment by tmansuroglu** [comment]
@Ammar-64 can you merge?

File: src/components/gamespage/GamesPageProgressSection/index.jsx [hunk 1]
```diff
@@ -11,6 +12,20 @@ const ProgressSection = ({
 }) => {
   const nextLevel = progressData.currentLevel + 1;
   const [t] = useTranslation();
+  const currentLanguage = i18next.language;
+  const gridSpanObj = {
```
Comment (louisrli) [inline src/components/gamespage/GamesPageProgressSection/index.jsx]: this is a constant object

**Review by louisrli — COMMENTED** [review]
no idea whats going on with the arabic adjustment code. write some comments

File: src/components/gamespage/GamesPageProgressSection/index.jsx (hunk 1 above)
Comment (tmansuroglu) [reply src/components/gamespage/GamesPageProgressSection/index.jsx]: PTAL

**Comment by tmansuroglu** [comment]
@louisrli added comments to arabic adjustment code

File: src/components/gamespage/GamesPageProgressSection/index.jsx [hunk 2]
```diff
@@ -1,16 +1,33 @@
 import React from "react";
 import "./index.css";
 import { useTranslation } from "react-i18next";
+import i18next from "i18next";
 import { Progress, Row, Col } from "antd";
 import { QuestionOutlined } from "@ant-design/icons";
 
+const GRID_SPAN_OBJ = {
+  currLevelSpan: 7,
```
Comment (louisrli) [inline src/components/gamespage/GamesPageProgressSection/index.jsx]: you dont need to add Span to the end of everything if the constant name already has it. we know these are spans

but you can just call this GRID_SPANS

and write a comment above it.

**Review by louisrli — APPROVED** [review]

File: src/components/gamespage/GamesPageProgressSection/index.jsx (hunk 2 above)
Comment (tmansuroglu) [reply src/components/gamespage/GamesPageProgressSection/index.jsx]: PTAL

**Comment by tmansuroglu** [comment]
@Ammar-64 can you merge?

**Pull request merged**